provision/core/mailhog/provision.sh

```diff
@@ -0,0 +1,53 @@
+
+mailhog_setup() {
+  if [[ -f "/etc/init/mailcatcher.conf" ]]; then
+    echo " * Cleaning up old mailcatcher.conf"
+    rm -f /etc/init/mailcatcher.conf
+  fi
+
+  if [[ ! -e /usr/local/bin/mailhog ]]; then
+    echo " * Installing MailHog"
+    curl --silent -L -o /usr/local/bin/mailhog https://github.com/mailhog/MailHog/releases/download/v1.0.0/MailHog_linux_amd64
+    chmod +x /usr/local/bin/mailhog
+  fi
+  if [[ ! -e /usr/local/bin/mhsendmail ]]; then
+    echo " * Installing MHSendmail"
+    curl --silent -L -o /usr/local/bin/mhsendmail https://github.com/mailhog/mhsendmail/releases/download/v0.2.0/mhsendmail_linux_amd64
+    chmod +x /usr/local/bin/mhsendmail
+  fi
+
+  if [[ ! -e /etc/systemd/system/mailhog.service ]]; then
+    echo " * Mailhog service file missing, setting up"
+    # Make it start on reboot
+    tee /etc/systemd/system/mailhog.service <<EOL
+[Unit]
```

Comment:
maybe extract this as file now that we have a mailhog folder so the script will be simplified?

Reply:
We could create a utility for this but we can't remove mailhog from VVV core in v3. We could deprecate and remove in v4.

Of all the things in VVV core installed by default, I think Mailhog is important enough to come out of the box

Reply:
For sure, I mean we can remove the code of that file from the bash script and put as file that is copied.
So it is simplified.

Reply:
Agreed

Reply:
Done